modules/core/test/unit/bitgo.ts

```diff
@@ -279,13 +280,30 @@ describe('BitGo Prototype Methods', function() {
   });
 
   describe('ECDH sharing secret', () => {
+    function getKey(seed: string) {
+      return bitcoin.HDNode.fromSeedBuffer(
+          crypto.createHash('sha256').update(seed).digest()
+      ).keyPair;
+    }
+
     it('should calculate a new ECDH sharing secret correctly', () => {
-      const bitgo = new TestBitGo();
-      const eckey1 = bitcoin.ECPair.makeRandom({ network: bitcoin.networks[common.getNetwork()] });
-      const eckey2 = bitcoin.ECPair.makeRandom({ network: bitcoin.networks[common.getNetwork()] });
-      const sharingKey1 = bitgo.getECDHSecret({ eckey: eckey1, otherPubKeyHex: eckey2.getPublicKeyBuffer().toString('hex') });
-      const sharingKey2 = bitgo.getECDHSecret({ eckey: eckey2, otherPubKeyHex: eckey1.getPublicKeyBuffer().toString('hex') });
-      sharingKey1.should.equal(sharingKey2);
+      for (let i=0; i<256; i++) {
+        const bitgo = new TestBitGo();
+        const eckey1 = getKey(`${i}.a`);
+        const eckey2 = getKey(`${i}.b`);
+        const sharingKey1 = bitgo.getECDHSecret({ eckey: eckey1, otherPubKeyHex: eckey2.getPublicKeyBuffer().toString('hex') });
+        const sharingKey2 = bitgo.getECDHSecret({ eckey: eckey2, otherPubKeyHex: eckey1.getPublicKeyBuffer().toString('hex') });
+        sharingKey1.should.equal(sharingKey2);
+
+        switch (i) {
```

Comment:
Why assert the first two keys? Do these match the prior implementation (if not, pending shares would need to be invalidated)

Reply:
Ok, I reviewed the ordering of the commits and since the test was modified first, I'm assuming the new ECDH implementation is entirely compatible with the old one. Approving based on that assumption

Reply:
I was just making sure here that the result is actually different every round. 

(It would be a nice feature if CI actually for every commit, not just the last one)